webapp/src/main/webapp/src/app/sender/archive/sender-archive.ts

```diff
@@ -0,0 +1,22 @@
+import {BaseEntity} from "../../shared/base-entity";
+
+export class SenderSyncArchive extends BaseEntity {
+
+	tableName?: string;
+
+	identifier?: string;
+
+	operation?: string;
+
+	messageUuid?: string;
+
+	requestUuid?: string;
+
+	status?: string;
+
+	dateSent?: string;
+
+	eventDate?: string;
+
```

Comment:
There is also **dateReceivedByReceiver**

Reply:
Please make sure you have pulled the latest changes in master then and add it, isn't there a requirement to show it?

Reply:
Done